src/test/java/core/basesyntax/StorageStrategyTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.StorageStrategy;
+import core.basesyntax.service.StorageSupplyService;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.StorageStrategyImpl;
+import core.basesyntax.service.impl.StorageSupplyServiceImpl;
+import core.basesyntax.strategy.AdditionHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.SubtractionHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageStrategyTest {
+    private static Storage localStorage;
+    private static StorageSupplyService localStorageSupply;
+    private static OperationStrategy operationStrategy;
+    private static StorageStrategy storageStrategy;
+
+    @BeforeClass
+    public static void setUp() {
+        Map<String, OperationHandler> operationHandlers = new HashMap<>();
+        operationHandlers.put("s", new AdditionHandler());
+        operationHandlers.put("b", new AdditionHandler());
+        operationHandlers.put("r", new AdditionHandler());
+        operationHandlers.put("p", new SubtractionHandler());
+        localStorage = new Storage();
+        localStorageSupply = new StorageSupplyServiceImpl(localStorage);
+        operationStrategy = new OperationStrategyImpl(operationHandlers);
+        storageStrategy = new StorageStrategyImpl(operationStrategy);
+    }
+
+    @Before
+    public void clean() {
+        localStorage.flush();
+    }
+
+    @Test
+    public void storageStrategyWorks_Ok() {
+        List<String[]> twoFruits = new ArrayList<>();
```

Comment:
```suggestion
        List<String[]> fruits = new ArrayList<>();
```
the same for the method below